pkg/lint/rules.go

```diff
@@ -375,6 +375,24 @@ func ValidateDuplicateColumnNames(ctx context.Context, p *pipeline.Pipeline, ass
 	return issues, nil
 }
 
+func ValidateInvalidPythonModuleName(ctx context.Context, p *pipeline.Pipeline, asset *pipeline.Asset) ([]*Issue, error) {
+	var issues []*Issue
+
+	if asset.Type == "python" {
+		parentDirs := strings.Split(filepath.Dir(asset.DefinitionFile.Path), "/")
```

Comment:
I'd use executable file path instead of the definition, while they are mostly the same, they don't have to be.

Reply:
can you please explain ? what is `executable file path`